src/types/any.rs

```diff
@@ -43,6 +44,15 @@ impl crate::AsPyPointer for PyAny {
     }
 }
 
+impl crate::IntoPyPointer for PyAny {
+    #[inline]
+    fn into_ptr(self) -> *mut ffi::PyObject {
+        let pointer = self.as_ptr();
+        mem::forget(self);
+        pointer
+    }
+}
```

Comment:
I don't think this impl is necessary and it's also not sound - moving a PyAny like this onto the stack and then converting it to a pointer is just creating a dangling pointer. Please remove.

```suggestion
```